tox.ini

```diff
@@ -15,7 +15,7 @@ sitepackages = false
 commands_pre =
   poetry install --all-extras
 commands =
-  pytest -o 'addopts=--cov --cov-config .coveragerc --cov-report term --cov-report xml --cov-report html' tests/
+  pytest -vv -o 'addopts=--cov --cov-config .coveragerc --cov-report term --cov-report xml --cov-report html' tests/
```

Comment:
Is this intentionally in the PR? It looks unrelated.

Reply:
I used this for debugging tests, just forgot to remove it.